src/core/congestion_control.h

```diff
@@ -13,10 +14,28 @@ typedef struct QUIC_ACK_EVENT {
 
     uint64_t LargestPacketNumberAcked;
 
+    uint64_t LargestPacketNumberSent;
+
     uint32_t NumRetransmittableBytes;
 
     uint32_t SmoothedRtt;
 
+    uint32_t MinRttSample;
+
+    uint32_t AdjustedAckTime;
+
+    uint64_t TotalBytesAcked;
```

Comment:
You need to comment this to make it clear what the difference is between this and the NumRetransmittableBytes field. And probably the two names should be more similar.

// Number of retransmittable bytes acked during the connection's lifetime